website/common/locales/en/messages.json

```diff
@@ -60,5 +60,8 @@
   "messageUserOperationProtected": "path `<%= operation %>` was not saved, as it's a protected path.",
   "messageUserOperationNotFound": "<%= operation %> operation not found",
   "messageNotificationNotFound": "Notification not found.",
-  "notificationsRequired": "Notification ids are required."
+
+  "notificationsRequired": "Notification ids are required.",
+
+  "bannedWordUsed": "Oops! Looks like this post contains a swearword, religious oath, or reference to an addictive substance or adult topic. Habitica has users from all backgrounds, so we keep our chat very clean. Feel free to edit your message so you can post it!"
```

Comment:
Can this be moved to groups.json, underneath `communityGuidelinesRead2` (about line 23)? That will keep related texts together which will help us in future when reviewing or changing them. It might help the translators too if they can see similar texts at the same time.